build_tools/bazel_to_cmake/bazel_to_cmake_targets.py

```diff
@@ -48,6 +48,9 @@
     "@llvm-project//mlir:StandardToSPIRVConversions": [
         "MLIRStandardToSPIRVTransforms"
     ],
+    "//third_party/llvm/llvm-project/mlir:GPUToVulkanTransforms": [
```

Comment:
I think this should be

```suggestion
    "@llvm-project//mlir:GPUToVulkanTransforms": [
```

Reply:
Anyway, this seems like a target that should be handled correct by default, with no explicit mapping required.

Reply:
Thanks for catching that, I got mixed up in the path when moving my changes to github. 
I do need this since there is a small inconsistency between bazel file and CMake file in the name of the library bazel uses "To" while Cmake "to" in the GPUToVulkanTransforms name.

Reply:
Yeah, just saw Geoffrey's comment, I simply missed that.